core/src/main/java/hivemall/tools/list/UDAFToOrderedList.java

```diff
@@ -406,6 +406,11 @@ void merge(@Nonnull List<Object> o_keyList, @Nonnull List<Object> o_valueList) {
 
             @Nonnull
             Pair<List<Object>, List<Object>> drainQueue() {
+                if (queueHandler == null) {
+                    return Pair.of((List<Object>) new ArrayList<Object>(),
+                        (List<Object>) new ArrayList<Object>());
```

Comment:
better to return `null` following the way in other SQL standard functions.

`terminate()`, `terminatePartial()`, and `@Nonnull` annotation in `drainQueue` also need to be changed.

Reply:
@myui Updated. PTAL!